src/builtin/tables.scrbl

```diff
@@ -47,53 +67,150 @@ my-table = table: name :: String, age :: Number, favorite-color :: String
 end
   }
 
-Note that @pyret{my-table} is referred to in many of the following examples.
+Entering the name of a table in the interactions window after running the program
+or using the @pyret{print} function will display a formatted version of the table.
```

Comment:
Calling `print(my-table)` will result in the output `<table>`.  Then, because `print` currently returns the value you passed it (paging @shriram!), CPO will render that value.  So you'll _actually_ see `<table>` followed by the rich rendering of that table.  But `print` had nothing to do with it.